compiler/surface/lexer_pl.cppo.ml

```diff
@@ -20,18 +20,18 @@
 
 #define MS_SCOPE "zakres"
 #define MS_CONSEQUENCE "konsekwencja"
-#define MS_DATA "data"
-#define MS_DEPENDS "zalezy od"
-#define MR_DEPENDS "zalezy", space_plus, "od"
+#define MS_DATA "dane"
+#define MS_DEPENDS "zależy od"
```

Comment:
So each time you have a `MS_XXXX` that contains an non-ASCII character you should define just below a `MR_XXXX` line with a format like `"pieni", 0x0105, "dze"` where you concat ASCII strings and UFT8 hex encoding of the non-ASCII characters. You can use sites like https://www.compart.com/en/unicode/U+0105 to figure out the encoding of the letter you're putting. This is annoying but required for the lexer, we tried to find a better way but this is the least annoying solution.